roles/sft-server/templates/sftd.unit.j2

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Running service: SFT server
+
+After=nginx.service
+Wants=nginx.service
+
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+
+# maximum size of core files (bytes)
+LimitCORE=infinity
+LimitCORESoft=0
+
+User={{ SFTD_USER }}
+Group={{ SFTD_GROUP }}
```

Comment:
You can just set `DynamicUser=true`, and hardcode the sftd user and group.

Reply:
Implemented.